### PR TITLE
update OS reporting measure

### PR DESCRIPTION
**Pull request by brianlball**
remove tests related to Xcel EDA Reporting and QAQC measure

### CODE REVIEW

File: openstudio-workflow.gemspec [hunk 1]
```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'ci_reporter_rspec', '~> 1.0.0'
   s.add_development_dependency 'coveralls', '~> 0.8.21'
   s.add_development_dependency 'json-schema', '~> 2.8.0'
+  s.add_development_dependency 'openstudio-standards', '~> 0.2.12'
```
Comment (nllong) [inline openstudio-workflow.gemspec:30]: yuck, this is an unfortunate dependency to add.

File: spec/files/output_request_osw/output_request.osw [hunk 2]
```diff
@@ -26,9 +26,6 @@
         },
         {
             "measure_dir_name": "openstudio_results"
-        },
-        {
-            "measure_dir_name": "Xcel EDA Reporting and QAQC"
```
Comment (nllong) [inline spec/files/output_request_osw/output_request.osw:31]: thanks for removing this

**Review by nllong — APPROVED** [review]
crazy number of files added. Was openstudio_results really that simple before?

File: openstudio-workflow.gemspec (hunk 1 above)
Comment (brianlball) [reply openstudio-workflow.gemspec:30]: yep... bc of the openstudio results measure for testing

Comment (nllong) [reply openstudio-workflow.gemspec:30]: blargh... okay

**Pull request merged**